### PR TITLE
feat(ecam): packs off abnormally warning

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #[issue_no]

## Summary of Changes

When maximum takeoff performance is required, it is possible to do a packs off takeoff. It is normal to takeoff with the packs turned off, and turn them on upon reaching the thrust reduction altitude. If you forget to do so the cabin will not remain pressurised when climbing to higher flight levels. My general feeling from the community is that this does happen quite frequently since we actually simulate the pressurisation system now. This PR adds the ECAM warning that exists to prevent such an occurence.

The exact logic requires some additional system implementation that is out of scope for this PR, so a few of the conditions have been approximated for now. The whole thing will later be replaced with the new FWS, but I felt this was still worth a small amount of effort in the short term. 

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Do a packs off takeoff and 60 seconds after passing 1500 feet you should get an upper ECAM warning as shown in the screenshot above... the lower ECAM will automatically switch to the BLEED page. The status page will show PACKS 1 + 2 as INOP. Turn one of the packs on and ensure the warning for that pack goes away, repeat for the other.

Do a packs off takeoff with both engine bleeds off. The particular warnings shown above should not occur.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by davidwalschots — APPROVED** [review]
🚀

**Review by beheh — APPROVED** [review]

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #6650
Tier of Testing  : 1
Date             : 06/02/2022

Testing Process:
Tested at EHEH and SKBO 
Packs off takeoff - ECAM watning and STS packs inop. OK
Bleed + packs off takeoff - No warning. OK

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -138,6 +138,8 @@
 1. [HYD] Simplified max step loop updater - @Crocket63
 1. [MISC] Fix for bleed/hyd system failure in some conditions - @Crocket63
```
Comment (beheh) [inline .github/CHANGELOG.md:140]: Please remove the extra line here.

**Review by beheh — CHANGES_REQUESTED** [review]

File: .github/CHANGELOG.md (hunk 1 above)
Comment (tracernz) [reply .github/CHANGELOG.md:140]: Oops. Good catch. :)

**Review by beheh — APPROVED** [review]

**Pull request merged**